config/packages/doctrine.yaml

```diff
@@ -5,6 +5,9 @@ doctrine:
         # IMPORTANT: You MUST configure your server version,
         # either here or in the DATABASE_URL env var (see .env file)
         #server_version: '15'
+        mapping_types:
+            new_enum: string
```

Comment:
dit lijkt een verder niet gerelateerde wijziging, maar dit is om die ENUMs in doctrine goed te laten werken vermoed ik?